ui/js/controllers/main.js

```diff
@@ -1,27 +1,63 @@
 "use strict";
 
 treeherderApp.controller('MainCtrl', [
-    '$scope', '$rootScope', '$routeParams', '$location', '$timeout',
+    '$scope', '$rootScope', '$routeParams', '$location', '$timeout', '$q',
     'ThLog', 'ThRepositoryModel', 'thPinboard', 'thTabs', '$document',
-    'thClassificationTypes', 'thEvents', '$interval', '$window',
+    'thClassificationTypes', 'thEvents', '$interval', '$window', 'thNotify',
     'ThExclusionProfileModel', 'thJobFilters', 'ThResultSetStore',
-    'thDefaultRepo', 'thJobNavSelectors', 'thTitleSuffixLimit',
+    'thDefaultRepo', 'thJobNavSelectors', 'thTitleSuffixLimit', '$http',
     function MainController(
-        $scope, $rootScope, $routeParams, $location, $timeout,
+        $scope, $rootScope, $routeParams, $location, $timeout, $q,
         ThLog, ThRepositoryModel, thPinboard, thTabs, $document,
-        thClassificationTypes, thEvents, $interval, $window,
+        thClassificationTypes, thEvents, $interval, $window, thNotify,
         ThExclusionProfileModel, thJobFilters, ThResultSetStore,
-        thDefaultRepo, thJobNavSelectors, thTitleSuffixLimit) {
+        thDefaultRepo, thJobNavSelectors, thTitleSuffixLimit, $http) {
         var $log = new ThLog("MainCtrl");
 
         // Query String param for selected job
         var QS_SELECTED_JOB = "selectedJob";
 
+        //var revisionPollInterval = 60000 * 5;
+        var revisionPollInterval = 15000;
```

Comment:
You should probably pick one of these and run with it. :) Please also provide a comment saying what it is.

I would say probably every 15 minutes would be fine.